openshift/settings.sh

```diff
@@ -1,4 +1,4 @@
-export PROJECT_NAMESPACE="devex-von"
+export PROJECT_NAMESPACE="8ad0ea"
```

Comment:
is this intentional? I thought we left the settings.sh file as is and edited the settings.local.sh

Reply:
Yes, the namespaces are different in OCP4